sanic/app.py

```diff
@@ -101,7 +101,8 @@ def decorator(listener):
         return decorator
 
     # Decorator
-    def route(self, uri, methods=frozenset({'GET'}), host=None):
+    def route(self, uri, methods=frozenset({'GET'}), host=None,
```

Comment:
Just a thought but later on in the future if we want to add more arguments to these (and in particular the shorthand versions) we may just want to start using `*args` and `**kwargs` to bubble them up to the `add_route` function without having to change them in 10 different places